micro_epsilon_scancontrol_driver/include/micro_epsilon_scancontrol_driver/driver.h

```diff
@@ -74,7 +80,19 @@ namespace scancontrol_driver
             void ServiceInvertX(
                 const std::shared_ptr<std_srvs::srv::SetBool::Request> request,
                 std::shared_ptr<std_srvs::srv::SetBool::Response> response);
-    
+            void ServiceSetExposureTime(
+                const std::shared_ptr<micro_epsilon_scancontrol_msgs::srv::SetTime::Request> request,
```

Comment:
preference: Adding aliases for these log type would make this more readable.

Reply:
Done